### DISCUSSION TITLE
Application Server error: lua_package_path

### DISCUSSION THREAD

**Discussion by mSerendipity** [5G-MAG/Getting-Started#72] [Q&A] [answered]
Hello, when I was trying to run 5gms-application-server, an error occurred:
ERROR:NginxWebProxy:nginx: [emerg] unknown directive "lua_package_path" in /tmp/rt_5gms_as.conf:48
![AS Error-:lua_package_path](https://github.com/5G-MAG/Getting-Started/assets/82158989/9b563429-3773-401f-aa37-19d4e806b830)

I've tried many times but still couldn't solve it. I've already installed OpenResty.
Can you please help me take a look? Thank you very much!

**Comment by dsilhavy** [comment, marked as answer]
Do you have a separate installation of nginx on your system? I might help to de-install the separate nginx installation as openresty installs nginx with the lua package.

**Reply by mSerendipity** [reply]
Thank you soooooo much! The problem is solved.
At first I think there's only OpenResty, because when I use the command 'which nginx', it only shows OpenResty:
![nginx](https://github.com/5G-MAG/Getting-Started/assets/82158989/3c784429-25f1-4558-925c-4203f48addf5)

But later I search nginx in the folder and find there's another nginx in usr/sbin, I delete it and run again but this time the error is there is no nginx. So I move the OpenResty's nginx to the usr/sbin folder and this time I run it successfully. It's still confusing because I've change the system path to point the nginx to OpenResty but it still use the other one. But since the problem is solved this doubt is not important. Thank you again!!!
